### ISSUE TITLE
[Request] Protect last X used tabs

### ISSUE THREAD

**Issue by dreamflasher** [rNeomy/auto-tab-discard#99]
Please add an option to not discard the last X tabs. LRU style.

**Comment by dumblob** [comment]
This is actually a disguised **way bigger topic** - see from https://github.com/deanoemcke/thegreatsuspender/issues/136#issuecomment-339794343 onwards.

**Comment by dreamflasher** [comment]
I don't think it needs to be blown up that much, but yeah of course you can build a rule engine around it, with different scheduling algorithms… make it turing complete ;) 
Is there any tab unloader there which supports LRU?

**Comment by anohren** [comment]
I was hoping for this option as well.
